packages/angular_devkit/build_optimizer/src/transforms/prefix-functions.ts

```diff
@@ -62,12 +68,17 @@ export function findTopLevelFunctions(parentNode: ts.Node): ts.Node[] {
     // We need to check specially for IIFEs formatted as call expressions inside parenthesized
     // expressions: `(function() {}())` Their start pos doesn't include the opening paren
     // and must be adjusted.
-    if (isIIFE(node)
+    if (
```

Comment:
First line goes with the `if`, last parenthesis with the last line, with 4 spaces of indentation:

```typescript
if (isIIFE(node)
    && previousNode.kind === ts.SyntaxKind.ParenthesizedExpression
    && node.parent
    && !hasPureComment(node.parent)) {
  topLEvelFunctions.push(node.parent);
} else if ((node.kind === ts.SyntaxKind.CallExpression || node.kind === ts.SyntaxKind.NewExpression)
    && !hasPureComment(node)) {
  topLevelFunctions.push(node);
}
```